bin/sof

```diff
@@ -40,6 +40,17 @@ class SofCommand < Thor
     end
     YAML.load(manifest)
   end
+
+  def self.start(given_args = ARGV, config = {})
+    super(given_args, config)
+  rescue Interrupt
```

Comment:
Should we rescue this within the runner so that we can print any results we have managed to collect?